src/index/updater/inscription_updater.rs

```diff
@@ -284,6 +344,26 @@ impl<'a, 'db, 'tx> InscriptionUpdater<'a, 'db, 'tx> {
     }
   }
 
+  fn calculate_sat(
```

Comment:
merge with what is in `update_inscription_location`

Reply:
Canada, Québec